static/admin/config.yml

```
@@ -41,24 +41,24 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "First name", name: "firstName", widget: "string" }
-      - { label: "Last name", name: "lastName", widget: "string" }
+      - { label: "First name", name: "firstName", widget: "string", pattern: ['.{2,20}', "Must have 2-20 characters"] }
+      - { label: "Last name", name: "lastName", widget: "string", pattern: ['.{2,20}', "Must have 2-20 characters"] }
       - { label: "Avatar image", name: "avatar", widget: "image" }
-      - { label: "Job position", name: "position", widget: "string" }
+      - { label: "Job position", name: "position", widget: "string", pattern: ['.{2,30}', "Must have 2-30 characters"] }
       - { label: "Phone number", name: "phone", widget: "string", required: false }
-      - { label: "Email", name: "email", widget: "string" }
-      - { label: "City of residence", name: "city", widget: "string" }
-      - { label: "Country of residence", name: "country", widget: "string" }
+      - { label: "Email", name: "email", widget: "string", pattern: ['/^.+@.+\..+$/', "Must be a valid e=mail address"] }
```

```suggestion
      - { label: "Email", name: "email", widget: "string", pattern: ['/^.+@.+\..+$/', "Must be a valid email address"] }
```
this character is unnecessary 😄

---

ooohh... fixed now :)